### PR TITLE
Move all possible logs to syslog

### PR DESCRIPTION
**Pull request by dpoirier**
And forward to papertrail

This will stop mail-bombing admins (me) when something goes wrong
with Ona.  We can set up alerts in papertrail that'll only trigger
every so often.

### CODE REVIEW

**Comment by dpoirier** [comment]
See also https://github.com/theirc/CTS/pull/19

File: conf/salt/project/newrelic_webmon/init.sls [hunk 1]
```diff
@@ -27,7 +27,7 @@ newrelic_ini_worker:
     - mode: 440
     - template: jinja
     - context:
-      log_file: "{{ vars.log_dir }}/newrelic-python-agent.log"
+       log_file: "stderr"
```
Comment (vkurup) [inline conf/salt/project/newrelic_webmon/init.sls]: looks like there's an extra leading space there.

**Comment by vkurup** [comment]
There are a few more references to our `log_dir`. Can these be removed?
```
./conf/salt/project/_vars.sls:17:{% set log_dir = path_from_root('log') %}
./conf/salt/project/web/app.sls:12:log_dir_{{ instance }}:
./conf/salt/project/web/app.sls:14:    - name: {{ vars.log_dir }}/{{ instance }}
./conf/salt/project/web/app.sls:20:      - file: log_dir
./conf/salt/project/dirs.sls:16:log_dir:
./conf/salt/project/dirs.sls:18:    - name: {{ vars.log_dir }}
```

**Comment by dpoirier** [comment]
I was unable to find a way to send the newrelic app monitor's logs to
syslog, so we still need each instance's log dir as a place for those to
go.  Too bad.

File: conf/salt/project/newrelic_webmon/init.sls (hunk 1 above)
Comment (dpoirier) [inline conf/salt/project/newrelic_webmon/init.sls]: On Mon, Nov 2, 2015 at 3:01 PM, Vinod Kurup <notifications@github.com>
wrote:

> looks like there's an extra leading space there.


​Fixed.​

**Comment by dpoirier** [comment]
Any other comments?

**Comment by vkurup** [comment]
Oops, sorry, meant to :+1: this one before.

**Pull request merged**